### PR TITLE
Prepare Release v0.75.0

### PR DESCRIPTION
**Pull request by VineethReddy02**
Resolves: #1624

### CODE REVIEW

File: README.md
Comment (yuriolisa) [inline README.md:1]: Could you please remove the last version on that table?

File: CHANGELOG.md
Comment (yuriolisa) [inline CHANGELOG.md:1]: Is it missing the Java-autoinstrumentation PR?

**Review by yuriolisa — COMMENTED** [review]
Could you also re-run the chloggen in order to get all these changes in place?

File: CHANGELOG.md
Comment (VineethReddy02) [reply CHANGELOG.md:1]: I don't see a java-inst PR being merged post-0.74.0 release.

**Comment by VineethReddy02** [comment]
@yuriolisa I have re-run the `make chlog-update`, and I see the same output, updated the PR by dropping the last version from the supportability matrix.

**Review by jaronoff97 — APPROVED** [review]

**Pull request merged**